### DISCUSSION TITLE
Auto refresh on 'Explore the data' and 'Design your dashboard'

### DISCUSSION THREAD

**Discussion by QihaMarz** [BSd3v/Dashboard-Helper#6] [Q&A] [answered]
Hi there!

I would like to ask why if I upload a file in 'Explore the data', it will be gone when I refresh meanwhile the 'Design your dashboard' it will be there forever unless I upload a new one.

Really appreciate your answer.

Thank you.

**Comment by BSd3v** [comment, marked as answer]
Hello @QihaMarz,

This is because the design page has need for the dcc.Store to be loaded in order to cater a saved layout. 

The home page made sense at the time to be memory. But you can change it however you would like.

**Reply by QihaMarz** [reply]
Thank you so much! So if I would like to change the design page so it can clear previous data after refreshed, I need to modify dcc.Store right?

**Reply by BSd3v** [reply]
Correct, just change it from "local" to "memory"
